Composer/packages/client/src/pages/botProject/RootBotExternalService.tsx

```diff
@@ -341,14 +330,33 @@ export const RootBotExternalService: React.FC<RootBotExternalServiceProps> = (pr
   };
 
   return (
-    <CollapsableWrapper title={formatMessage('External services')} titleStyle={title}>
+    <Fragment>
+      <div css={title}>{formatMessage('Language Understanding')}</div>
+      <div css={subtext}>
+        {formatMessage.rich(
+          '<a>Language Understanding Intelligent Service (LUIS)</a> is a machine learning-driven recognition service that enables advanced conversational capabilities. If you already have LUIS keys you’d like to use, you can paste them below. To fetch existing keys from Azure or create new keys, you can click “Get LUIS keys”. <a2>Learn more</a2>.',
```

Comment:
I wonder if you can pass props to the tags inside of `formatMessage` and then pass them to the link? Might be worth trying.

```suggestion
          '<a href="...">Language Understanding Intelligent Service (LUIS)</a> is a machine learning-driven recognition service that enables advanced conversational capabilities. If you already have LUIS keys you’d like to use, you can paste them below. To fetch existing keys from Azure or create new keys, you can click “Get LUIS keys”. <a href="...">Learn more</a>.',
```

Reply:
Doesnt seem to work. Shows as invalid FormatMessage syntax :/ That would be cool as then we could write the Link function once and have the params passed down to further abstract formatted text with links.